nixos/extractors/services.nix

```diff
@@ -45,6 +46,15 @@ in {
           else "${config.services.esphome.address}:${toString config.services.esphome.port}";
       };
 
+      firefox-syncserver = mkIf config.services.firefox-syncserver.enable ({
+          name = "Firefox Syncserver";
+          icon = "services.firefox-syncserver";
+          details.listen.text = "0.0.0.0:${toString config.services.firefox-syncserver.settings.port}";
```

Comment:
The default listen host seems to be 127.0.0.1:

https://github.com/NixOS/nixpkgs/blob/4cba8b53da471aea2ab2b0c1f30a81e7c451f4b6/nixos/modules/services/networking/firefox-syncserver.nix#L308

Reply:
Oh never mind! I was looking at the nginx proxy option.

Reply:
So probably you want to replace 0.0.0.0 with `${settings.host or "127.0.0.1"}`

Reply:
Agree, thanks for the feedback - updated commit to https://github.com/oddlama/nix-topology/pull/7/commits/8892a50f280172dfc6adeee1560755c3a691b5ea which implements your recommendations :grin: